ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/section_subjdomain_numbers_and_numeric_structures.scs

```diff
@@ -8,21 +8,10 @@ section_subjdomain_numbers_and_numeric_structures
 => nrel_idtf:
 	[Раздел базы знаний IMS, описывающий числа и числовые структуры]
 	(* <- lang_ru;; *);
-=> nrel_introduction:
-	...
-(*
-	<= nrel_sc_text_translation:
-	 ...
-	 (*
-		 -> rrel_example:
-		   "file://content_html/introduction_section_subjdomain_numbers.pdf"
-			(* <- lang_ru;;	*);;
-	 *);;
-*);
```

Comment:
What was wrong with that part? It is important, why was it deleted?

Reply:
We still need it.